managed/devops/bin/yb_backup.py

```diff
@@ -1610,6 +1612,30 @@ def delete_bucket_obj(self):
         else:
             self.run_program(del_cmd)
 
+    def find_nfs_storage(self):
```

Comment:
As I had mentioned in the our initial discussion, we already allow for running commands in parallel, so there is no point in running these commands sequentially. Check how the `find_data_dirs` function is utilized by the script and do something similar for finding the NFS storage too.

Reply:
Done. Thanks!